src/posix/platform/hdlc_interface.cpp

```diff
@@ -600,6 +600,20 @@ int HdlcInterface::OpenFile(const Url::Url &aRadioUrl)
         {
             tios.c_cflag |= CRTSCTS;
         }
+        else
+        {
+#ifndef __APPLE__
+            int flags;
+#endif // __APPLE__
```

Comment:
nit: we only add the comment when >20 lines away from the matching statement.

```suggestion
#endif
```

Reply:
ok